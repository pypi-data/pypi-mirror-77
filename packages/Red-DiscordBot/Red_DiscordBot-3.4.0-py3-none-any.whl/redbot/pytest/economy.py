import pytest
from redbot.core import bank as bank_module

__all__ = ["bank"]


@pytest.fixture()
def bank(config, monkeypatch):
    from redbot.core import Config

    with monkeypatch.context() as m:
        m.setattr(Config, "get_conf", lambda *args, **kwargs: config)
        # noinspection PyProtectedMember
        bank_module._init()
        return bank_module
