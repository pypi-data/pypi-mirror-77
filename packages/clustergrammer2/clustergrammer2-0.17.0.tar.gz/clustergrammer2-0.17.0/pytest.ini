[pytest]
testpaths = clustergrammer2/tests examples
norecursedirs = node_modules .ipynb_checkpoints
# addopts = --nbval --current-env
