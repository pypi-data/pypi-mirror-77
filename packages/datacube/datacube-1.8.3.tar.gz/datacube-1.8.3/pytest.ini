[pytest]
addopts = --doctest-modules
testpaths = datacube tests integration_tests
norecursedirs = .* build dist .git tmp*
filterwarnings = ignore::FutureWarning

