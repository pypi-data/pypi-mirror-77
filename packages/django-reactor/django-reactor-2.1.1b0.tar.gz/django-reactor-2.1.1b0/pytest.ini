[pytest]
python_files = test*.py
addopts = --ff -v --nomigrations --doctest-modules tests/
DJANGO_SETTINGS_MODULE = fision.settings
