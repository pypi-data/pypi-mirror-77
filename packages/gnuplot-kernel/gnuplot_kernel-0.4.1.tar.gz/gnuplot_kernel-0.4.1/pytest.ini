[pytest]
pyargs = gnuplot_kernel
doctest_optionflags = ALLOW_UNICODE ALLOW_BYTES NORMALIZE_WHITESPACE
