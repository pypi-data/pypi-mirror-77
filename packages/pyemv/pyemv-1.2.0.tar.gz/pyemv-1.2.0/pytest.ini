[pytest]
addopts = --ignore=docs --doctest-modules
doctest_optionflags= NORMALIZE_WHITESPACE
