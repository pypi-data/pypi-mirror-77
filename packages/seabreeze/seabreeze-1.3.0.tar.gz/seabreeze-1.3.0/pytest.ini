[pytest]
addopts = -v -rs -p no:faulthandler
