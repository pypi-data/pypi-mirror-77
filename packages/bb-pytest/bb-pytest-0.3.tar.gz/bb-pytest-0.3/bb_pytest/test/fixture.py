import pytest


def test_test1():
    pass


@pytest.mark.failure
def test_failure1():
    assert False


def test_test2():
    pass


@pytest.mark.skipped
def test_skipped1():
    pytest.skip("")


def test_test3():
    pass


@pytest.mark.skipped
def test_skipped2():
    pytest.skip("")


def test_test4():
    pass


@pytest.mark.failure
def test_failure2():
    assert False


@pytest.mark.failure
def test_failure3():
    assert False

xfail = pytest.mark.xfail

@xfail(reason="bug")
def test_xpass():
    assert False


@pytest.mark.failure
@xfail(reason="bug fixed")
def test_xfail():
    pass


@pytest.mark.slowtest
def test_marked():
    pass
