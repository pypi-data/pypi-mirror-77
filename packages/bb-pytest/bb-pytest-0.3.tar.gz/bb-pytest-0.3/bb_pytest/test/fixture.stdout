============================= test session starts ==============================
platform linux2 -- Python 2.7.3 -- pytest-2.3.4
plugins: xdist
collected 9 items

../bb_pytest/test/fixture.py .F.s.s.FF

=================================== FAILURES ===================================
________________________________ test_failure1 _________________________________

    def test_failure1():
>       assert False
E       assert False

../bb_pytest/test/fixture.py:9: AssertionError
________________________________ test_failure2 _________________________________

    def test_failure2():
>       assert False
E       assert False

../bb_pytest/test/fixture.py:33: AssertionError
________________________________ test_failure3 _________________________________

    def test_failure3():
>       assert False
E       assert False

../bb_pytest/test/fixture.py:37: AssertionError
================ 3 failed, 4 passed, 2 skipped in 0.02 seconds =================
